src/pages/Embed.js

```diff
@@ -103,73 +104,83 @@ const Embed = () => {
     useEffect(() => {
         fetchInstanceInfo();
         // eslint-disable-next-line react-hooks/exhaustive-deps
+        const fullPath = window.location.search.substring(1);
+        const qArray = fullPath.split('=');
+        if (qArray[0] === 'space') {
+          setUrl(qArray[1])
+        }
```

Comment:
grabs the space from the iframe url that exists in the [snapshot app](https://github.com/ECWireless/snapshot/blob/2dfe16d597350dae95e76e29cdc317837b2921d7/src/views/Proposal.vue#L150)